subcommands/keys/ca_show.go

```diff
@@ -26,6 +29,9 @@ func init() {
 	}
 	caCmd.AddCommand(cmd)
 	cmd.Flags().BoolVarP(&prettyFormat, "pretty", "", false, "Display human readable output of each certificate")
+	justShowFlags.Add(cmd, "just-root", "Only show the Factory root CA certificate")
+	justShowFlags.Add(cmd, "just-tls", "Only show the device-gateway TLS certificate")
+	justShowFlags.Add(cmd, "just-cas", "Only show device authenticate certificates trusted by the device-gateway")
```

Comment:
maybe, `just-ca` or even better `just-device-ca`?

Reply:
i was on the fence. including `device` kind of make sense. It's technically plural, so it felt weird not doing that and then seeing multiple certs. Let me add `device` because that does make it more consistent.

Reply:
updated in force-push